modules/aioseop_sitemap.php

```diff
@@ -1434,6 +1530,18 @@ function do_notify() {
 				'bing'   => 'https://www.bing.com/webmaster/ping.aspx?siteMap=',
 			);
 
+			/**
+			 * Filter to update the sitemap ping URLs.
+			 *
+			 * @since ?
+			 *
+			 * @param array $notify_url {
+			 *      Array of search engine name vs. the URL to ping.
+			 *
+			 *      @type   string  Name of the search engine.
```

Comment:
The array was documented incorrectly

```
/**
 * Filter to update the sitemap ping URLs.
 *
 * @since ?
 *
 * @param array $notify_url {
 *     Array of search engine name vs. the URL to ping.
 *
 *     @type string $bing   URL for Bing Sitemap ping.
 *     @type string $google URL for Google Sitemap ping.
 * }
 */
```